docs/readme.ptxt

```diff
@@ -88,42 +71,29 @@ There are three ways to get the latest stable release:
     OpenSFX manually. This is really not that difficult as it may
     sound, so do not worry too much about that.
       o Download location:
-          http://mz.openttdcoop.org/bundles/opensfx/releases/
+          https://www.openttd.org/downloads/opensfx-releases/latest.html
       o Installation instructions:
           Installing OpenSFX Manually.
 
-  * If you already have OpenTTD up and running using the original
-    Tranport Tycoon Deluxe files, the ingame Online Content service is
-    the easy way to obtain OpenSFX.
+  * If you already have OpenTTD up and running, the in-game Online Content
+    service is the easy way to obtain OpenSFX.
       o Download location:
-          use the ingame Online content service
+          use the in-game Online content service
       o Installation instructions:
           Installing OpenSFX using the Online Content service.
 
-  * Just like OpenTTD, there are nightly builds of OpenSFX available
-    as well. Every evening around 18:18 CE(S)T a new build of OpenSFX
-    is created automatically (if there is something new that is).
-    Unlike stable releases these builds are not tested to see if they
-    work, but if a nightly does not work, it does not break anything
-    either. Keep a stable or working nightly sitting in the /data dir
-    and just delete a broken nightly to get OpenSFX working again.
-      o Download location:
-          http://mz.openttdcoop.org/bundles/opensfx/nightlies/
-      o Installation instructions:
-          Installing OpenSFX Manually.
-
 2.1) Installing OpenSFX Manually
 ==== ===========================
 1. First, make sure that you have downloaded and installed at least
    OpenTTD version 1.0.0 or a recent nightly.
 2. Next, download the latest OpenSFX package. (stable nightly)
-3. Unpack the zip into the OpenTTD's /data directory. There is no
+3. Unpack the zip into the OpenTTD's /baseset directory. There is no
    need to unpack the tar file, so just leave it as it is. Your
-   OpenTTD /data directory is either located in:
+   OpenTTD /baseset directory is either located in:
       * An OpenTTD folder in your user account's home directory:
           o Windows: C:\Documents and Settings\<username>\My Documents\OpenTTD
```

Comment:
These days this is ```C:\Users\<username>\Documents\OpenTTD``` - if you're updating the others, it might be as well to update this too.